src/si/energy.rs

```diff
@@ -47,6 +47,15 @@ quantity! {
         @milliwatt_hour: 3.6_E0; "mW · h", "milliwatt hour", "milliwatt hours";
         @microwatt_hour: 3.6_E-3; "µW · h", "microwatt hour", "microwatt hours";
 
+        @petaelectronvolt: 1.602_177_E-4; "PeV", "petaelectronvolt", "petaelectronvolts";
+        @teraelectronvolt: 1.602_177_E-7; "TeV", "teraelectronvolt", "teraelectronvolts";
+        @gigaelectronvolt: 1.602_177_E-10; "GeV", "gigaelectronvolt", "gigaelectronvolts";
+        @megaelectronvolt: 1.602_177_E-13; "MeV", "megaelectronvolt", "megaelectronvolts";
+        @kiloelectronvolt: 1.602_177_E-16; "KeV", "kiloelectronvolt", "kiloelectronvolts";
+        @hectoelectronvolt: 1.602_177_E-17; "heV", "hectoelectronvolt", "hectoelectronvolts";
+        @decaelectronvolt: 1.602_177_E-18; "daeV", "decaelectronvolt", "decaelectronvolts";
+        @electronvolt: 1.602_177_E-19; "eV", "electronvolt", "electronvolts";
```

Comment:
* Can you adjust the conversion factor for `electronvolt` to `1.602_176_634_E-19` and adjust the other units accordingly (https://physics.nist.gov/cgi-bin/cuu/CCValue?evj|ShowSecond=electronvolt&First=hev). This more exact value may come from the recent 2019 SI redefinition.
 * Correct the `kiloelectronvolt` abbreviation to `keV`.

Reply:
Sorry for late response. Fixed the conversion factor and KeV -> keV.